src/executables/uber-time.js

```diff
@@ -8,13 +8,19 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 const service = new CommandExecutionService();
 
+const address = program.args[0].trim().toString();
```

Comment:
Also, I think this will cause two linting issues

![image](https://user-images.githubusercontent.com/8136030/27316113-8b560dc8-554b-11e7-80f2-658fbd301bcd.png)

Also, I'm not sure this will succeed without a `program.parse(process.argv)` call that precedes it. Mind double-checking?